test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -32,6 +34,13 @@ private async Task<HttpResponseMessage> SendOutputRequest(string functionName, I
             return await this.SendGetRequest(requestUri);
         }
 
+        private async Task<HttpResponseMessage> SendOutputPostRequest(string functionName, string query)
+        {
+            string requestUri = $"http://localhost:{this.Port}/api/{functionName}";
+
+            return await this.SendPostRequest(requestUri, query);
```

Comment:
compiler not letting me to do that, I should also update the return type, complier error -> "Since this is an async method, the return expression must be of type 'HttpResponseMessage' rather than 'Task<HttpResponseMessage>'"

Reply:
Shouldn't we wait until the response for the tests? Having async and awaiting makes sense to me, why do you think that's not needed?

Reply:
You're still returning the task. The caller should be awaiting that task. Functionally this should not change anything.

Reply:
ohh from the compiler error, I thought I also need to update it return HttpResponseMessage instead of Task along with removing the async! Gotcha, thanks.